ubi8-openjdk-8.yaml

```diff
@@ -19,9 +19,11 @@ labels:
 - name: "com.redhat.component"
   value: "openjdk-8-ubi8-container"
 - name: "usage"
-  value: "https://access.redhat.com/documentation/en-us/openjdk/8/html/using_openjdk_8_source-to-image_for_openshift/index"
+  value: &docs "https://access.redhat.com/documentation/en-us/openjdk/8/html/using_openjdk_8_source-to-image_for_openshift/index"
```

Comment:
Should probably point to the upstream docs too?

Reply:
Woops, fixed now. Thanks for spotting